hyp3_gamma/dem.py

```diff
@@ -31,17 +30,14 @@ def get_dem_features() -> Generator[ogr.Feature, None, None]:
     del ds
 
 
-def intersects_dem(geometry: ogr.Geometry) -> bool:
-    for feature in get_dem_features():
-        if feature.GetGeometryRef().Intersects(geometry):
-            return True
-
-
 def get_dem_file_paths(geometry: ogr.Geometry) -> List[str]:
     file_paths = []
     for feature in get_dem_features():
         if feature.GetGeometryRef().Intersects(geometry):
             file_paths.append(feature.GetField('file_path'))
+    if not file_paths:
+        file_paths.append('/vsicurl/https://copernicus-dem-30m.s3.amazonaws.com/'
+                          'Copernicus_DSM_COG_10_N00_00_E106_00_DEM/Copernicus_DSM_COG_10_N00_00_E106_00_DEM.tif')
```

Comment:
Open to suggestions on how to handle the "no dem tiles" case.  I think we can effectively hand any non-intersecting or zero-filled geotiff/vrt to the gdal.Warp command and get all-zero output.

[gdal_create](https://gdal.org/programs/gdal_create.html) could be a good option if/when we upgrade to gdal>=3.2.0